src/__tests__/expand-test.js

```diff
@@ -16,17 +16,24 @@ describe('Expanding style objects', () => {
     ).toMatchSnapshot()
   })
 
+  it('should not throw on undefined values', () => {
+    expect(
+      expand({
+        border: undefined,
+        padding: null,
+      })
+    ).toMatchSnapshot()
+  })
```

Comment:
The same as in https://github.com/robinweser/inline-style-expand-shorthand/pull/11.